hardware_interface/include/hardware_interface/types/hardware_interface_type_values.hpp

```diff
@@ -17,8 +17,13 @@
 
 namespace hardware_interface
 {
+/// Constant defining position interface
 constexpr const auto HW_IF_POSITION = "position";
+/// Constant defining velocity interface
 constexpr const auto HW_IF_VELOCITY = "velocity";
+/// Constant defining acceleration interface
+constexpr const auto HW_IF_ACCELERATION = "acceleration";
+/// Constant defining effort interface
```

Comment:
so we had this discussion I believe many times now, but wasn't the idea to have these constants as part of the msg package?

Reply:
Yes and now.
My reasoning against the msg package is reduction of dependencies. 
As proposed here, one need only one simple header in a package that is already dependency.
Otherwise, we introduce additional ones.
Also, this functionality is not something message-related. Only argument for it would be convenience (but more dependencies...).

Reply:
> looks good to me, even though I'd like to see these values then applied to other packages as well - for consistency.

I will do that, as soon as we clarify exact names.